src/main/java/lotto/LottoGenerator.java

```diff
@@ -2,23 +2,40 @@
 
 import java.util.Collections;
 import java.util.List;
+import java.util.function.Function;
+import java.util.function.UnaryOperator;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
+/**
+ * LottoGenerator 는 로또 자동생성시 번호를 전달해 줄 수 있다.
+ */
 public class LottoGenerator {
+    private static final List<LottoNumber> LOTTO_NUMBERS;
+
+    private static final int SEED = 1;
+    private static final UnaryOperator<Integer> INCREMENT_FUNC = i -> i + 1;
     private static final int TOTAL_LOTTO_NUMBER_COUNT = 45;
-    private static final int DEFAULT_SELECT_COUNT = 6;
+    private static final int DEFAULT_CHOICE_COUNT = 6;
 
-    public static List<LottoNumber> autoMode() {
-        final List<LottoNumber> allLottoNumbers = Stream.iterate(1, i -> i + 1)
+    /**
+     * 1부터 45까지 모든 LottoNumber 를 생성한다.
+     */
+    static {
+        LOTTO_NUMBERS = Stream.iterate(SEED, INCREMENT_FUNC)
                 .limit(TOTAL_LOTTO_NUMBER_COUNT)
                 .map(LottoNumber::new)
                 .collect(Collectors.toList());
```

Comment:
```suggestion
        LOTTO_NUMBERS = IntStream.rangeClosed(1, 45)
                .mapToObj(LottoNumber::new)
                .collect(Collectors.toList());
```
이렇게 바꿔보면 어떨까요? 조금 더 간결하게 코드를 만들 수 있을 것 같아요. :)

Reply:
이 부분이 정말 필요했었는데 알려주셔서 감사합니다🙏